src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowUpdate.cs

```diff
@@ -30,6 +32,14 @@ public sealed class RowUpdate : RowEditBase
         private const string UpdateScript = "UPDATE {0} SET {1} {2}";
         private const string UpdateScriptMemOptimized = "UPDATE {0} WITH (SNAPSHOT) SET {1} {2}";
         private const string SelectStatement = "SELECT {0} FROM {1}";
+        private string ValidateUpdateOnlyOneRow = "DECLARE @numberOfRows int = 0;" + Environment.NewLine +
```

Comment:
should this be `const`? If not, I think `validateUpdateOnlyOneRow` is the correct name casing.